app/views/groups/show.html.erb

```diff
@@ -4,13 +4,20 @@
 <h1>
   <%= @group.name %>
   <%= @group.description %>
+  <%= @group.kind %>
+  <% @leaders.each do |leader| %>
```

Comment:
@nikitr Do you need the `@leaders` variable here? Doesn't `@group.users.leaders` work?